### PR TITLE
Feature complete registration email

### PR DESCRIPTION
**Pull request by avoevodin**
But i haven't figured out why the static for admin panel doesn't work.
#32

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,17 +33,23 @@ python3 -m smtpd -n -c BrigadierSMTPDServer localhost:1025
 * Create the .env file. Example:
 ```shell
 cat > .env << __EOF__
-POSTGRES_DB=brigadier
+PPOSTGRES_DB=brigadier
```
Comment (iliadmitriev) [inline README.md:36]: double `P` PPOSTGRES

**Comment by avoevodin** [comment]
> But i haven't figured out why the static for admin panel doesn't work. #32

and i don't understand, why coverage checks pragma no cover lines in app.py. In local tests i have 100% coverage.

**Comment by iliadmitriev** [comment]
> > But i haven't figured out why the static for admin panel doesn't work. #32
> 
> and i don't understand, why coverage checks pragma no cover lines in app.py. In local tests i have 100% coverage.

try to create `.coveragerc` configuration https://coverage.readthedocs.io/en/coverage-5.5/config.html

**Comment by iliadmitriev** [comment]
could you check and make a reference here which line is not covered?
as far as I understood it's 
https://github.com/avoevodin/brigadier/blob/feature_complete_registration_email/brigadier/worker/app.py#L21

File: brigadier/worker/app.py [hunk 2]
```diff
@@ -18,14 +18,10 @@
     broker_url = f'pyamqp://{RABBIT_USER}:{RABBIT_PASS}@' \
         f'{RABBIT_HOST}:{RABBIT_PORT}/{RABBIT_VHOST}'
 else:
-    broker_url = None
+    broker_url = None  # pragma: no cover
```
Comment (iliadmitriev) [inline brigadier/worker/app.py:21]: why are you disabling this line from coverage count?
do you consider, there's no possible way to test it?

**Review by iliadmitriev — CHANGES_REQUESTED** [review]
resolve comments above

File: brigadier/accounts/tests.py [hunk 3]
```diff
@@ -24,15 +26,17 @@ def test_create_user_public_group_exists(self):
...
+                reverse('accounts:registration'),
+                {
+                    'username': 'User_name',
+                    'email': 'user@example.com',
+                    'password1': '1234',
+                    'password2': '1234',
+                }
+            )
         self.assertEqual(response.status_code, 302)
         self.assertEqual(response.url, reverse('accounts:registration_done'))
```
Comment (iliadmitriev) [inline brigadier/accounts/tests.py:41]: you've created a mock for `worker.email.tasks.send_verification_mail.delay` but you don't check was it called during account registration

File: brigadier/accounts/tests.py [hunk 4]
```diff
@@ -46,15 +50,17 @@ def test_create_user_public_group_does_not_exists(self):
...
```
Comment (iliadmitriev) [inline brigadier/accounts/tests.py:65]: you've created a mock for `worker.email.tasks.send_verification_mail.delay` but you didn't check was it called during account registration

**Review by iliadmitriev — CHANGES_REQUESTED** [review]
another 2 comments

**Review by iliadmitriev — APPROVED** [review]

**Pull request merged**